app/controllers/converts_controller.rb

```diff
@@ -28,6 +28,9 @@ def create
       filename: file_name,
       layout: false
     )
+  rescue ActionController::ParameterMissing
+    flash.now[:alert] = "Missing 'file' or 'format' form option."
```

Comment:
I'm not familiar with `flash.now`, what does this do differently from `flash[:alert] = 'blah'`?

Also, as a user will I see this error message if I submit a form without choosing a file? If so, the error message should be more friendly.